docs/_static/custom.css

```diff
@@ -89,10 +89,22 @@ div.caution {  /* csslint allow: overqualified-elements */
     background-color: #f0ad4e;
 }
 
+div.caution pre {  /* csslint allow: overqualified-elements */
+    background-color: #a6844e;
+}
+
 div.important {  /* csslint allow: overqualified-elements */
     background-color: #d9534f;
 }
 
+div.important pre {  /* csslint allow: overqualified-elements */
+    background-color: #b84e4f;
+}
+
+div.tip {  /* csslint allow: overqualified-elements */
```

Comment:
A CSS rule already exists for it, just next rule to it! So, Why have you defined a new rule for `tip div` element?

Reply:
I guess you want to define `div.tip pre` rule, right?

Reply:
oops! yessss, thanks for telling me 😥